docs/user_guide/storing/index_in_memory.md

```diff
@@ -33,10 +33,16 @@ doc_index.index(docs)
 doc_index = InMemoryExactNNIndex[MyDoc](docs)
 ```
 
-Additionally, you can preserve your index as a binary file and instantiate a new one using this file:
+Alternatively, you can pass an `index_file_path` argument to make sure that the index can be restored if persisted from that specific file.
 ```python
 # Save your existing index as a binary file
-doc_index.persist('docs.bin')
+docs = DocList[MyDoc](MyDoc() for _ in range(10))
+
+doc_index = InMemoryExactNNIndex[MyDoc](index_file_path='docs.bin')
+doc_index.index(docs)
+
+# or in one step:
```

Comment:
what do you mean by "# or in one step:" here?

Reply:
Initialize it with the docs inserted